db-service/lib/cqn2sql.js

```diff
@@ -289,7 +289,7 @@ class CQN2SQLRenderer {
       .map(c => c.sql)
 
     this.entries = [[JSON.stringify(INSERT.entries)]]
-    return (this.sql = `INSERT INTO ${entity}${alias ? ' as ' + this.quote(alias) : ''} (${
+    return (this.sql = `INSERT INTO ${this.quote(entity)}${alias ? ' as ' + this.quote(alias) : ''} (${
```

Comment:
I'm not in favour of this inflationary use of the quite expensive `this.quote` function → we should plan a separate review on that